osgar/tools/lidarview.py

```diff
@@ -220,7 +268,8 @@ def lidarview(gen, caption_filename, callback=False):
     skip_frames = 0
     frames_step = 0
 
-    history = History(gen)
+    #history = History(gen)
```

Comment:
I am not sure about the legacy generator :( ... I would clean this up, somehow ...

Reply:
Well, so far I have tried to make the changes as non-invasive as possible. But I think we need to cut inside a bit more to be able to introduce new features using the indexed reader.